tests/logger/LogHelper.test.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable max-len */
-import {GraphQLError} from 'graphql'
+import {GraphQLError,
+    Kind} from 'graphql'
```

Comment:
I think we should update the formatter, this looks really weird to me.

Reply:
What's your proposal on how to format this line?

Reply:
I reformated it with multiline = true option. Does it look better now?

Reply:
Yes, much better 👍